Composer/packages/client/src/constants.ts

```diff
@@ -8,112 +8,6 @@ export const BASEURL = `${process.env.PUBLIC_URL || ''}/api`;
 
 //the count about the undo/redo
 export const UNDO_LIMIT = 10;
-/**
```

Comment:
Where did all this go?

Reply:
@srinaath asked if I could remove the actions in my PR. They should have been moved in removed with the recoil changes.

Reply:
Yep. Those constants are no longer required. They were part of the context changes